app/views/claims/teacher_reference_number.html.erb

```diff
@@ -3,19 +3,18 @@
     <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
 
     <%= form_for current_claim, url: claim_path do |form| %>
-      <h1 class="govuk-heading-xl">
-        <%= form.label :teacher_reference_number, t("tslr.questions.teacher_reference_number")  %>
-      </h1>
+      <%= form_group_tag current_claim do %>
+        <h1 class="govuk-label-wrapper">
+          <%= form.label :teacher_reference_number, t("tslr.questions.teacher_reference_number"), class: "govuk-label govuk-label--xl" %>
+        </h1>
 
-      <span class="govuk-hint">This is on the certificate you got when you qualified as a teacher, or your school can tell you.</span>
+        <span class="govuk-hint">This is on the certificate you got when you qualified as a teacher, or your school can tell you.</span>
 
-      <div class="govuk-form-group">
-        <%= form.text_field :teacher_reference_number, spellcheck: "false", class: "govuk-input" %>
-      </div>
-
-      <div class="govuk-form-group">
-        <%= form.submit "Continue", class: "govuk-button" %>
-      </div>
+        <div class="govuk-form-group">
+          <%= form.text_field :teacher_reference_number, spellcheck: "false", class: css_classes_for_input(current_claim, :teacher_reference_number) %>
```

Comment:
Missing `errors_tag`?

Reply:
I must have messed something up. I can't have missed all these… worrying.

Reply:
Suggest looking through everywhere you've added the new tags and double checking?

Reply:
yep

Reply:
I think I have them all, the DOB one is a bit odd, I'll have to look at it tomorrow.